lib/Renard/Curie/Component/PageDrawingArea.pm

```diff
@@ -148,6 +164,8 @@ sub can_move_to_next_page {
 	$self->current_page_number < $self->document->last_page_number;
 }
 
+has scrolled_window => (is => 'rw');
```

Comment:
You should move this attribute up to the top with the other attributes.

Also, this attribute will still be undefined until you set it. I recommend doing that in setup_drawing_area:

```perl
$self->scrolled_window(  $scrolled_window );
```